docs/source/model_design/risk_factors.rst

```diff
@@ -137,14 +155,13 @@ has changed and all else has remained exactly the same.
   counterfactual scenario could be conceptualized as "what would the lung 
   cancer rate in the US be in 1954 *if no one smoked* and **all else was equal**?" 
   Then, we could evaluate the independent effect of smoking on lung cancer
-  without interference from any other factors.
+  without interference from any other factors. 
 
-However, by definition, the counterfactual scenario is impossible to directly 
-evaluate, which is the core challenge in causal inference. Luckily, there are 
-several strategies that can be used to attempt to indirectly answer the 
-counterfactual question and perform causal inference. Some examples include
-randomized control trials and adjustment for confounding variables in 
-epidemiology studies.
+  However, we cannot wave a magic wand and explore the lung cancer rate in 
+  this alternative universe in 1954 to answer this question. Luckily, there 
+  are several methods in the field of epidemiology that attempt to answer 
+  conterfactual questions regarding causal relationships through randomized 
+  controlled trials and other types of studies. 
 
```

Comment:
The section starts off by talking about causal inference, then counter-factual, then causal inference again using the bradford hill criteria. The counterfactual paragraph feels slightly out of place. Maybe it can be called out separately in a note box?